common/lib/capa/capa/customrender.py

```diff
@@ -137,3 +136,33 @@ def get_html(self):
         return xhtml
 
 registry.register(TargetedFeedbackRenderer)
+
+#-----------------------------------------------------------------------------
+
+
+class ClarificationRenderer(object):
+    """
+    A clarification appears as an inline [?] which reveals more information when the user
```

Comment:
s/[?]/icon/

this is clearer and allows us to change the icon while preserving the correctness of the docstring.